src/python/pants/core/goals/test.py

```diff
@@ -396,6 +561,11 @@ class EnvironmentAware:
             """
         ),
     )
+    batch_size = IntOption(
+        advanced=True,
+        default=128,
+        help=style_batch_size_help(uppercase="Tester", lowercase="tester"),
+    )
```

Comment:
How about leaving this part of the PR out for now and adding in a follow up? That way we can properly consider the right way to do batching for test, e.g. what default to use. And we can make sure the `help message is good etc

Reply:
Works for me 👍 I can hard-code a value where this field is currently being used for now